test/integration/notice_test.rb

```diff
@@ -0,0 +1,30 @@
+require 'test_helper'
+
+class NoticeGovspeakTest < ActionDispatch::IntegrationTest
```

Comment:
There's already a notice_test.rb in the test/components/ directory, it contains one test to check that nothing is rendered if no parameters are passed, which the integration test can't check. Could I combine the files? Would that work/be sensible?